mobile/src/components/common/input/input.tsx

```diff
@@ -1,11 +1,13 @@
-import React, { ReactElement } from 'react';
-import { TextInput } from 'react-native';
+import React, { ReactElement, useEffect, useState } from 'react';
+import { TextInput, ViewStyle as UIViewStyle } from 'react-native';
```

Comment:
@Liza-Veis Is it possible to import TextInput component from ~/components/common/common like View or Text ?

Reply:
No, we only need to re-export from there components that we use throughout the app. We will import specific components directly from the library. We won't need TextInput somewhere else, because we have an Input component for this and we want to use it instead

Reply:
Should I then use Input instead of TextInput in Search Component?

Reply:
No, Search Component will have another style and logic